src/main/kotlin/kotliquery/Session.kt

```diff
@@ -129,6 +154,26 @@ open class Session(
         return rows(query, extractor).toList()
     }
 
+    fun batchManyStatements(statements: Collection<String>): List<Int> {
```

Comment:
This is not something we use, but is supported by jdbc. So I thought I'd add those, just for completeness. I have no problems with changing the name. Since this function does not take any query params, it implies it does not use prepared statements, but begin explicit is better.

We can of course just leave this out. I have never used such statements in real work work, and can be easily added if someone else would like it...